common/djangoapps/third_party_auth/api/views.py

```diff
@@ -91,7 +91,8 @@ def do_get(self, request, identifier):
         # In the future this can be trivially modified to return the inactive/disconnected providers as well.
 
         return Response({
-            "active": active_providers
+            "active": active_providers,
+            "account_active": user.is_active, #rebase-ironwood -custom work- YONK-1532
```

Comment:
I think there is no need to add this comment as this custom code was added before ironwood.